src/components/ContextualMenu/ContextualMenuDropdown/ContextualMenuDropdown.tsx

```diff
@@ -55,14 +56,18 @@ export type Props<L = null> = {
  */
 const getPositionStyle = (
   position: Position,
+  verticalPosition: VerticalPosition,
```

Comment:
In Tooltip we have combined positions of `btm-left`, `top-left`, `left`, etc. Maybe we should stick to the combined way of positioning? To stay backward compatible `left` can be the same as `btm-left`.

Reply:
I did think about it, but decided against it to keep things simple. I'll try and give that a go.

Reply:
I did think about it, but decided against it to keep things simple. I'll try and give that a go.

Reply:
Having looked at it I'd rather treat this as a future improvement, I don't want to break the existing logic and it's already quite complex. To do this right it would need major refactoring.

As is, vertical position will be automatically adjusted and displayed at the bottom (which is as it works currently).